src/components/article/articleCard/index.js

```diff
@@ -27,6 +27,10 @@ export class ArticleCard extends Component {
                 <i className="fa fa-comments-o" /> 
                 {" "}
                 {24}
```

Comment:
What is the relevance of this number `24`

Reply:
This 24 was included when creating the article, am not sure how it is being used but I think it was put to show the number of comments.

Reply:
hey @AmosWels should I remove the 24

Reply:
Yes you should remove it. I don't see it's actual relevance. Rather it'll confuse users

Reply:
Okay, let me do that

Reply:
Hey, @AmosWels am done.